switchbot/devices/evaporative_humidifier.py

```diff
@@ -0,0 +1,193 @@
+import logging
+from typing import Any
+
+from bleak.backends.device import BLEDevice
+
+from ..const import SwitchbotModel
+from ..const.evaporative_humidifier import (
+    TARGET_HUMIDITY_MODES,
+    HumidifierMode,
+    HumidifierWaterLevel,
+)
+from ..models import SwitchBotAdvertisement
+from .device import SwitchbotEncryptedDevice
+
+_LOGGER = logging.getLogger(__name__)
+
+COMMAND_HEADER = "57"
+COMMAND_GET_CK_IV = f"{COMMAND_HEADER}0f2103"
+COMMAND_TURN_ON = f"{COMMAND_HEADER}0f430101"
+COMMAND_TURN_OFF = f"{COMMAND_HEADER}0f430100"
+COMMAND_CHILD_LOCK_ON = f"{COMMAND_HEADER}0f430501"
+COMMAND_CHILD_LOCK_OFF = f"{COMMAND_HEADER}0f430500"
+COMMAND_AUTO_DRY_ON = f"{COMMAND_HEADER}0f430a01"
+COMMAND_AUTO_DRY_OFF = f"{COMMAND_HEADER}0f430a02"
+COMMAND_SET_MODE = f"{COMMAND_HEADER}0f4302"
+
+MODES_COMMANDS = {
+    HumidifierMode.HIGH: "010100",
+    HumidifierMode.MEDIUM: "010200",
+    HumidifierMode.LOW: "010300",
+    HumidifierMode.QUIET: "010400",
+    HumidifierMode.TARGET_HUMIDITY: "0200",
+    HumidifierMode.SLEEP: "0300",
+    HumidifierMode.AUTO: "040000",
+}
+
+
+class SwitchbotEvaporativeHumidifier(SwitchbotEncryptedDevice):
+    """Representation of a Switchbot Evaporative Humidifier"""
+
+    def __init__(
+        self,
+        device: BLEDevice,
+        key_id: str,
+        encryption_key: str,
+        interface: int = 0,
+        model: SwitchbotModel = SwitchbotModel.EVAPORATIVE_HUMIDIFIER,
+        **kwargs: Any,
+    ) -> None:
+        self._force_next_update = False
+        super().__init__(device, key_id, encryption_key, model, interface, **kwargs)
+
+    @classmethod
+    async def verify_encryption_key(
+        cls,
+        device: BLEDevice,
+        key_id: str,
+        encryption_key: str,
+        model: SwitchbotModel = SwitchbotModel.EVAPORATIVE_HUMIDIFIER,
+        **kwargs: Any,
+    ) -> bool:
+        return await super().verify_encryption_key(
```

Comment:
I'm not sure this will work without implementing `get_basic_info`

https://github.com/sblibs/pySwitchbot/blob/94882d3f611e3d015f98385a36e21e2a0ea1daf8/switchbot/devices/device.py#L813

Reply:
Hmm, you are right. The problem is that I don't know what would that command be for this device and what kind of data it should contain. This device doesn't have battery so looking to match that value is out the window. I'll try to come up with something.

Reply:
I'm going to be traveling again soon so I may miss GitHub notifications.  Feel free to reach out on Discord (same handle) if you don't hear back for a bit

Reply:
I think any command that works will probably be fine for now with a comment that we can replace it with something better later

Reply:
The latest push should take care of this